libs/@guardian/atoms-rendering/package.json

```diff
@@ -33,9 +33,9 @@
 	},
 	"peerDependencies": {
 		"@emotion/react": "^11.11.1",
-		"@guardian/commercial": "^10.0.0",
-		"@guardian/consent-management-platform": "^13.5.0",
-		"@guardian/libs": "^15.1.0",
+		"@guardian/commercial": "^10.5.0",
+		"@guardian/consent-management-platform": "^13.6.0",
+		"@guardian/libs": "^15.2.0",
```

Comment:
I believe that technically, any change in the peer dependencies would result in a major bump in the consuming library…

Reply:
Good point, rather than confuse things lets bump peer deps later.